app/lib/actions/katello/repository/sync.rb

```diff
@@ -35,13 +35,16 @@ def plan(repo, pulp_sync_task_id = nil, options = {})
           fail ::Katello::Errors::InvalidActionOptionError, _("Cannot skip metadata check on non-yum repositories.") if skip_metadata_check && !repo.yum?
 
           sequence do
-            # clear yum metadata if validate_contents is on (to avoid metadata corruption issues)
             plan_action(Pulp::Repository::RemoveUnits, :repo_id => repo.id, :content_unit_type => ::Katello::YumMetadataFile::CONTENT_TYPE) if validate_contents
-
-            sync_args = {:pulp_id => repo.pulp_id, :task_id => pulp_sync_task_id, :source_url => source_url, :options => pulp_sync_options}
-            output = plan_action(Pulp::Repository::Sync, sync_args).output
+            sync_args = {:smart_proxy_id => SmartProxy.pulp_master.id, :pulp_id => repo.pulp_id, :task_id => pulp_sync_task_id, :source_url => source_url, :options => pulp_sync_options}
```

Comment:
we don't need task_id at all anymore, nor in any of the actions related with syncing, want to remove it?  This was when we let pulp handle sync plans schedules.